### ISSUE TITLE
Migrate sealed-secrets chart to this repo

### ISSUE THREAD

**Issue by onedr0p** [bitnami/charts#4016]
Seeing as the official Helm chart repo will be deprecated on Nov 13th 2020, does it make sense to maintain this chart here instead?

https://github.com/helm/charts/tree/master/stable/sealed-secrets

Related: https://github.com/bitnami-labs/sealed-secrets/issues/389

**Comment by andresbono** [comment]
Hi @onedr0p, thanks for opening this issue and creating #4017. We will be looking into it.

**Comment by onedr0p** [comment]
Please feel free to add/remove maintainers as you see fit.

**Comment by andresbono** [comment]
@onedr0p, closing this issue in favor of https://github.com/bitnami-labs/sealed-secrets/issues/389

https://github.com/bitnami-labs/sealed-secrets/issues/389#issuecomment-717863252:
> TL;DR the sealed-secrets project is going to maintain the helm chart (one of the key motivator is to ensure it will be kept backwards compatible and stay in sync with the main release manifest)
